.github/workflows/lint_python.yml

```
@@ -0,0 +1,21 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell --skip="*/misspelling.json" || true  # --ignore-words-list=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=120 --show-source --statistics
```

```suggestion
      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=90 --show-source --statistics
```
In our code, we use a max line length of 90 cols :)

---

There are lines that are longer than 90 but we run this test in `--exit-zero` mode until we can clean the codebase.